### PR TITLE
Google-style Popup task

### PR DESCRIPTION
**Pull request by olek557**
### HTML & CSS practice: Google-style Popup

[Demo](https://olek557.github.io/html-css-popup/index.html) |
[Code base](https://github.com/olek557/html-css-popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/olek557/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+          </div>
+        </div>
+      </div>
+      <div class="nav__item">
+        <a href="/" class="user-menu">A</a>
+      </div>
+    </nav>
+  </header>
+</body>
+</html>
```
Comment (A-Ostrovnyy) [inline submissions/olek557/html-css-popup/index.html]: Add an empty line at the end of every file.

File: submissions/olek557/html-css-popup/style.css [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  margin-right: 30px;
+  position: relative;
+}
+
+.nav__item {}
```
Comment (A-Ostrovnyy) [inline submissions/olek557/html-css-popup/style.css]: Remove unused class/selectors.

File: submissions/olek557/html-css-popup/style.css [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+  padding: 0;
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.visually-hidden {
+  position: absolute !important;
```
Comment (A-Ostrovnyy) [inline submissions/olek557/html-css-popup/style.css]: Avoid using `!important` rule.

File: submissions/olek557/html-css-popup/style.css [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+  outline: none;
+  box-shadow: 0 0 5px 1px rgba(0, 0, 0, 0.6);
+}
+
+input:focus ~ label {
+  box-shadow: 0 0 5px 1px rgba(0, 0, 0, 0.6);
+}
+input:focus ~ .icon {
+  box-shadow: 0 0 5px 1px rgba(0, 0, 0, 0.6);
+
```
Comment (A-Ostrovnyy) [inline submissions/olek557/html-css-popup/style.css]: Remove unused lines.

File: submissions/olek557/html-css-popup/style.css [hunk 5]
```diff
@@ -0,0 +1,241 @@
...
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.visually-hidden {
+  position: absolute !important;
+  clip: rect(1px 1px 1px 1px); /* IE6, IE7 */
```
Comment (A-Ostrovnyy) [inline submissions/olek557/html-css-popup/style.css]: CSS Tricks recommend using `clip: rect(0 0 0 0);`.

**Comment by A-Ostrovnyy** [comment]
Nice work!

**Comment by olek557** [comment]
@A-Ostrovnyy Thank for comments!

**Pull request merged**